compiler/messages.h

```diff
@@ -259,11 +259,11 @@ static const char* warnmsg[] = {
     /*210*/ "possible use of symbol before initialization: \"%s\"\n",
     /*211*/ "possibly unintended assignment\n",
     /*212*/ "possibly unintended bitwise operation\n",
-    /*213*/ "tag mismatch\n",
+    /*213*/ "tag mismatch (expected \"%s\", got \"%s\")\n",
     /*214*/ "possibly a \"const\" array argument was intended: \"%s\"\n",
     /*215*/ "expression has no effect\n",
     /*216*/ "nested comment\n",
-    /*217*/ "loose indentation\n",
+    /*217*/ "mixed tabs and spaces used for indentation\n",
```

Comment:
Indeed I think this will trigger if two statements at the same "level" have different indentation. Maybe: "inconsistent indentation (did you mix tabs and spaces?)"

We could always split it into two warnings if there's no good wording.

Reply:
I like this idea.

Reply:
> Indeed I think this will trigger if two statements at the same "level" have different indentation. Maybe: "inconsistent indentation (did you mix tabs and spaces?)"

Yep, that's a lot better! Changed it to that.